include/cuco/detail/static_map_kernels.cuh

```diff
@@ -156,6 +156,64 @@ __global__ void insert(
   if (threadIdx.x == 0) { *num_successes += block_num_successes; }
 }
 
+template <std::size_t block_size,
+          typename InputIt,
+          typename atomicT,
+          typename viewT,
+          typename Hash,
+          typename KeyEqual>
+__global__ void erase(
+  InputIt first, InputIt last, atomicT* num_successes, viewT view, Hash hash, KeyEqual key_equal)
+{
+  using BlockReduce = cub::BlockReduce<std::size_t, block_size>;
+  __shared__ typename BlockReduce::TempStorage temp_storage;
+  std::size_t thread_num_successes = 0;
+
+  auto tid = block_size * blockIdx.x + threadIdx.x;
+  auto it  = first + tid;
+
+  while (it < last) {
+    if (view.erase(*it, hash, key_equal)) { thread_num_successes++; }
+    it += gridDim.x * block_size;
+  }
+
+  // compute number of successfully inserted elements for each block
+  // and atomically add to the grand total
+  std::size_t block_num_successes = BlockReduce(temp_storage).Sum(thread_num_successes);
+  if (threadIdx.x == 0) { *num_successes += block_num_successes; }
```

Comment:
This is going to do a `memory_order_seq_cst` `fetch_add`. Probably doesn't really impact perf very much, but this can/should be a `memory_order_relaxed`.